src/foam/nanos/crunch/lite/CapableAdapterDAO.js

```diff
@@ -0,0 +1,70 @@
+foam.CLASS({
+  package: 'foam.nanos.crunch.lite',
+  name: 'CapableAdapterDAO',
+  extends: 'foam.dao.AbstractDAO',
+
+  javaImports: [
+    'java.util.Arrays'
+  ],
+
+  documentation: `
+    Adapts a Capable object to the DAO interface.
+  `,
+
+  properties: [
+    {
+      name: 'capable',
+      class: 'FObjectProperty',
+      of: 'foam.nanos.crunch.lite.Capable'
+    },
+    // {
+    //   name: 'of',
```

Comment:
guess we also don't neeed the of as well since that wasn't the problem for  thee NPEs before?

Reply:
Yeah, also it's set in the code that constructs this now